keanu-python/keanu/tensor.py

```diff
@@ -59,3 +59,7 @@ def __infer_tensor_from_scalar(scalar):
         else:
             raise NotImplementedError("Generic types in a ndarray are not supported. Was given {}".format(type(scalar)))
 
+    @staticmethod
+    def _to_ndarray(java_tensor):
```

Comment:
As discussed offline, single underscore prefix is just by convention - it needs to be accessible by us in other modules but discouraged for users (since it takes java objects as a parameter).